src/ui/components/VPNToggle.qml

```diff
@@ -286,7 +286,7 @@ VPNButtonBase {
         anchors.fill: toggle
         hoverEnabled: (VPNController.state === VPNController.StateOn || VPNController.state === VPNController.StateOff)
         cursorShape: Qt.PointingHandCursor
-        onClicked: clicked()
+        onClicked: handleClick()
```

Comment:
The issue here is that handleClick() is called twice: the mouse-area and the button-base (line 26~). Is there a way to avoid it?

Reply:
yep- thank you for catching! fixed in d8429e7